Total Instances: 86

Total Correct: 57
Total Runtime Failed: 11
Total Compilation Failed: 5
Total Test Failed: 11
Total Infinite Loop: 2

Accuracy: 66.27906976744185
Runtime Rate: 12.790697674418606
Compilation Rate: 5.813953488372093
Test Failed Rate: 12.790697674418606
Infinite Loop Rate: 2.3255813953488373

=================================================================================================
Failed Test Files: ['codeforces_110_B.py', 'atcoder_ABC155_E.py', 'codeforces_622_A.py', 'codeforces_514_A.py', 'atcoder_ABC042_A.py', 'codeforces_58_B.py', 'atcoder_ARC062_B.py', 'atcoder_ABC136_B.py', 'codeforces_49_A.py', 'atcoder_AGC002_A.py', 'atcoder_ABC169_D.py'] 
Failed Test Details: ['Filename: codeforces_622_A.py Actual: 311 Generated: 311.0\n', 'Filename: codeforces_58_B.py Actual: 733035 \n244345 \n48869 \n1  Generated: 733035\n244345\n48869\n1\n', 'Filename: atcoder_ABC136_B.py Actual: 90909\n Generated: ', 'Filename: atcoder_AGC002_A.py Actual: Positive\n Generated: Negative\n', 'Filename: codeforces_110_B.py Actual: abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd Generated: abcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\nabcd\n', 'Filename: codeforces_49_A.py Actual: NO\n Generated: Enter a question: NO\n', 'Filename: atcoder_ABC042_A.py Actual: YES\n Generated: NO\n', 'Filename: codeforces_514_A.py Actual: 4221\n Generated: 4221.0\n', 'Filename: atcoder_ARC062_B.py Actual: 0\n Generated: -1\n', 'Filename: atcoder_ABC155_E.py Actual: 1\n Generated: ', 'Filename: atcoder_ABC169_D.py Actual: 10\n Generated: 3\n'] 
=================================================================================================
Runtime Error Files: ['atcoder_ABC114_C.py', 'atcoder_ARC102_C.py', 'codeforces_96_B.py', 'codeforces_306_A.py', 'atcoder_ABC135_D.py', 'codeforces_189_A.py', 'atcoder_ABC169_C.py', 'atcoder_ABC132_F.py', 'atcoder_ABC168_C.py', 'atcoder_ABC051_A.py', 'codeforces_334_A.py'] 
Runtime Error Details: ['Filename: atcoder_ABC135_D.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC135_D.py", line 22, in <module>\n    l = j - (ch - \'0\')\nTypeError: unsupported operand type(s) for -: \'str\' and \'str\'\n', 'Filename: codeforces_334_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/codeforces_334_A.py", line 22, in <module>\n    print(even.pop(even.size() - 1), end=" ")\nAttributeError: \'list\' object has no attribute \'size\'\n', 'Filename: atcoder_ABC169_C.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC169_C.py", line 3, in <module>\n    a = int(input())\nValueError: invalid literal for int() with base 10: \'814793613347395 1.99\'\n', 'Filename: atcoder_ABC168_C.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC168_C.py", line 3, in <module>\n    a = float(input())\nValueError: could not convert string to float: \'1 2 9 49\'\n', 'Filename: atcoder_ABC114_C.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC114_C.py", line 14, in <module>\n    S = input()\nEOFError: EOF when reading a line\n', 'Filename: codeforces_96_B.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/codeforces_96_B.py", line 28, in <module>\n    main()\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/codeforces_96_B.py", line 23, in main\n    value = int(s, 2)\nValueError: invalid literal for int() with base 2: \'74\'\n', 'Filename: atcoder_ARC102_C.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ARC102_C.py", line 16, in <module>\n    main()\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ARC102_C.py", line 11, in main\n    n = int(input("Enter the value of n: "))\nValueError: invalid literal for int() with base 10: \'1 200000\'\n', 'Filename: atcoder_ABC051_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 3, in <module>\n    a, b, c = sys.stdin.readline().split(\',\')\nValueError: too many values to unpack (expected 3)\n', 'Filename: codeforces_189_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/codeforces_189_A.py", line 3, in <module>\n    n = int(input())\nValueError: invalid literal for int() with base 10: \'29 27 18 2\'\n', 'Filename: codeforces_306_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/codeforces_306_A.py", line 6, in <module>\n    for i in range(m - int(n % m)):\nTypeError: \'float\' object cannot be interpreted as an integer\n', 'Filename: atcoder_ABC132_F.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/codellama/avatar/java/python-sanitized/atcoder_ABC132_F.py", line 14, in <module>\n    q = len.size()\nAttributeError: \'list\' object has no attribute \'size\'\n'] 
=================================================================================================
Compilation Error Files: ['atcoder_AGC007_C.py', 'codeforces_203_A.py', 'codeforces_569_A.py', 'codeforces_678_B.py', 'atcoder_AGC025_A.py'] 
=================================================================================================
Infinite Loop Files: ['codeforces_340_A.py', 'codeforces_669_A.py'] 
=================================================================================================
